config/file_config.go

```diff
@@ -9,6 +9,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/docker/go-units"
```

Comment:
I'm curious, are this units the same you'd use for k8s?

Reply:
I believe so -- it supports KGMT and a few more, with or without i and B, as well as plain numbers. So "GiB", "Tb", etc.